src/core/Chip/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Icon from '../Icon'
+import s from './styles'
+
+// Only allow lower case letters, optionally seperated by underscore
+const mdIconNamePattern = /^[a-z]+_*[a-z]+$/
```

Comment:
remove the regex, we'll simplify this by adding `type` to allow `Image` or `Icon` as in other place in code.